docs/arcaflow/workflows/schemas.md

```diff
@@ -39,18 +41,18 @@ Schemas that are not composed within an ObjectSchema do not have an `Object ID`.
 |---------------------------|---------------|-------------------------------|
 | Object                    | MyFirstObject | MyFirstObject                 |
 | List[Object]              | MyFirstObject | list_MyFirstObject            |
-| List[List[String]]        | MyFirstObject | list_list_MyFirstObject       |
-| List[List[List[String]]]  | MyFirstObject | list_list_list_MyFirstObject  |
+| List[List[Object]]        | MyFirstObject | list_list_MyFirstObject       |
+| List[List[List[Object]]]  | MyFirstObject | list_list_list_MyFirstObject  |
  
 ### Remaining Schemas
 
-* `ScopeSchema`s do not get a schema name.
+* `ScopeSchema`s do not use a schema name.
 * `RefSchema`s use the schema name of the type to which they point.
 
 
 ## Generated Combined Schema Names
 
-`bindConstants(list[any], any)` generates a new schema name from the schemas provided as arguments. Because the output of this function is a schema name and ID for a single list element, only the scheme name of the list's items is used. It is concatenated with a double underscore `__` with the schema name of the second argument.
+`bindConstants(list[any], any)` generates a new schema name from the schemas provided as arguments. Because the output of this function is a value for a single list element, only the schema name of the list's items is used. It is concatenated with a double underscore `__` with the schema name of the second argument.

```

Comment:
Do you really want to include the parameter types in the reference to `bindConstants()`?  It seems unnecessary, given that the function is not polymorphic, and it's kind of wordy (and you haven't done so for the other references to the function).

I think we should rework this text (the function doesn't generate the schema name; the output of the function is a value which is a list possibly of several items; etc.).  How about something along the lines of,
> The name of the schema for the value returned by a given call to `bindConstants()` is generated from the names of the schemas of the parameters to the call.  Because the output of this function is always a list, the `list_` prefix is omitted from the schema name, and only the schema name of the list's items is used. The name is formed by concatenating the name of the schema of the first parameter's list items with the name of the schema of the second parameter, separated by a double underscore `__`.

[I still think that we shouldn't omit the leading `list_` prefix from the final schema name -- I realize that it is understood to be there and that actually specifying it takes up space, but omitting it seems unnecessarily inconsistent since the item returned really is a list.]

Reply:
added the edit